tool/teleport/common/teleport.go

```diff
@@ -64,11 +70,12 @@ func Run(options Options) (executedCommand string, conf *service.Config) {
 	// define global flags:
 	var ccf config.CommandLineFlags
 	var scpFlags scp.Flags
+	var dumpFlags dumpFlags
 
 	// define commands:
 	start := app.Command("start", "Starts the Teleport service.")
 	status := app.Command("status", "Print the status of the current SSH session.")
-	dump := app.Command("configure", "Print the sample config file into stdout.")
+	dump := app.Command("configure", "Generate config file.")
```

Comment:
I would expand this a bit to explain what will be generated e.g. something like `Generate config file that will start Teleport auth, proxy and SSH services`.